### ISSUE TITLE
more details terminal output message

### ISSUE THREAD

**Issue by satrioaw** [stablekernel/aqueduct#126]
I tried running and get this result

```
Resolving dependencies... 
Got dependencies!
Starting...
Server running.
```
it will be more better with message which port running and which api got hit by client.

**Comment by joeconwaystk** [comment]
This information is available in the log file. The name of that file defaults to api.log. You can run the application with an attached shell and have output to the console. 

Leaving this issue open; to resolve, add support for a '--console' flag to the application start script that turns on console logging and does not detach the shell.

**Comment by joeconwaystk** [comment]
addressed in 2.0 branch
